test/libsolidity/smtCheckerTests/operators/bitwise_not_fixed_bytes.sol

```diff
@@ -1,7 +1,12 @@
 pragma experimental SMTChecker;
+
 contract C {
-  function f() public pure returns (byte) {
-    return (~byte(0xFF));
-  }
+	function f() public pure {
+		// ffff0000 in bytes4
+		bytes4 x = ~bytes4(hex"ffff");
+		assert(x == 0xffff0000); // fails
+		assert(x == 0x0000ffff); // holds
```

Comment:
Are the comments the expected outcomes or the current one, i.e. I suggest to rephrase to "should fail" and "should hold".